guides/common/modules/proc_using-ansible-job-templates-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Ansible_Job_Templates_for_Package_and_Errata_Actions{context}"]
+= Using Ansible job templates for package and errata actions
```

Comment:
Switching provider for REX jobs?

I'm not sure the actions are limited to packages and errata.
Can you try to find an umbrella term for the actions that you can change the provider for?